tvaultuiv2/src/views/private/certificates/components/CertificatesDashboard/index.js

```diff
@@ -652,7 +654,7 @@ const CertificatesDashboard = () => {
             <EditDeletePopper
               onDeleteClicked={() => onDeleteCertificateClicked(certificate)}
               onEditClicked={() => onEditListItemClicked(certificate)}
-              admin={contextObj.isAdmin}
+              admin
```

Comment:
Should it be hardcoded?
```suggestion
              admin
```